04/app.js

```diff
@@ -0,0 +1,27 @@
+//1 + 3
+const divList = document.querySelectorAll('div') 
+
+const addClass = function() {
+    const className = 'clicked'
+    const timeValue = parseInt(this.dataset.time)
+    setTimeout(() => this.classList.add(className), timeValue);  
```

Comment:
Można też użyć dodatkowej zmiennej zamiast funkcji strząłkowej tj. 
``
const self = this;
settTimeout(function() {
    self.classList.add(className)
}, timeValue);
```